install/packages.sh

```diff
@@ -73,6 +73,9 @@ sudo pacman -S --noconfirm --needed \
   omarchy-chromium \
   pamixer \
   pinta \
+  pipewire-alsa \
+  pipewire-jack \
+  pipewire-pulse \
```

Comment:
This seems like something that should just be in the aarch64 group, so we don't install this for everyone else.